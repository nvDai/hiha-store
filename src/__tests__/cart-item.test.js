import React from 'react'

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { CartItem } from '../components/cart-item/cart-item.component'

describe('CartItem', () => {
	it('should render without crashing', () => {
		const props = {
			item: {
				imageUrl: '',
				price: '',
				name: '',
				quantity: '',
			},
		}
		const wrapper = shallow(<CartItem {...props} />)

		expect(toJson(wrapper)).toMatchSnapshot()
	})
})
