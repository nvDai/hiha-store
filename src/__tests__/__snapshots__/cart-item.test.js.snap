// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem should render without crashing 1`] = `
<div
  className="cart-item"
>
  <img
    alt="items"
    src=""
  />
  <div
    className="item-details"
  >
    <span
      className="name"
    />
    <span
      className="price"
    >
       x $
    </span>
  </div>
</div>
`;
